backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/GenerateDeliveryDays.java

```diff
@@ -92,6 +93,11 @@ else if (PARAM_DateTo.equals(name))
 		{
 			p_tour = InterfaceWrapperHelper.create(getCtx(), getRecord_ID(), I_M_Tour.class, ITrx.TRXNAME_None);
 		}
+		if (I_M_TourVersion.Table_Name.equals(getTableName()) && getRecord_ID() > 0)
```

Comment:
please use the modern way of assigning parameters in this process (`de.metas.process.Param` annotation)

Reply:
Done!